### PR TITLE
MTM-61325 changelog, operationsCreated and operationsUpdated counters…

### PR DESCRIPTION
**Pull request by jkot-c8y**
… added to usage statistics.

Preview of new change-log entry (with hardcoded 2024-11-04 date):
image

Preview from gh-action (change-log not visible, because of no date)
https://cumulocity-iot-c8y-docs-build-pr-2601.surge.sh/enterprise-tenant/usage-and-billing/

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+date:
+title: Added operation request counters to usage statistics
+product_area: Platform services
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+component:
+  - value: component-OG_650_b2
+    label: Core platform
```
Comment (jkot-c8y) [inline content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]: I have found also component:
```
  - value: component-JlFdtOPva
    label: REST API
```
used in other change-log entries, but it is less used in 10.20 release. We can decide as a team which one fits better here.

**Review by karol-c8y — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-61325
+version: 10.20.611.0
+---
+Two new properties: `operationsCreatedCount`, `operationsUpdateCount` were added to all REST endpoints returning usage statistics detailed requests counters.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]:
```suggestion
Two new properties `operationsCreatedCount`, `operationsUpdateCount` have been added to all REST endpoints which will return detailed request counters for usage statistics. 
```

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-61325
+version: 10.20.611.0
+---
+Two new properties: `operationsCreatedCount`, `operationsUpdateCount` were added to all REST endpoints returning usage statistics detailed requests counters. 
+At the first step of this new feature {{< product-c8y-iot >}} returns those values always equal `0`.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]:
```suggestion
In a first implementation step, the {{< product-c8y-iot >}} platform returns these properties with `0` values. 
```

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-61325
+version: 10.20.611.0
+---
+Two new properties: `operationsCreatedCount`, `operationsUpdateCount` were added to all REST endpoints returning usage statistics detailed requests counters. 
+At the first step of this new feature {{< product-c8y-iot >}} returns those values always equal `0`. 
+Implementation of counting logic in reaction to REST requests and MQTT messages creating and updating operations will be added in the following changes to the 10.20 release.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]:
```suggestion
In a future implementation, counting logic in reaction to REST requests and MQTT messages creating and updating operations will be added.  
```

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md (hunk 1 above)
Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]: I would use REST API here to be more precise.

Comment (jkot-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]: done, squashed in first commit

File: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+component:
+  - value: component-JlFdtOPva
+    label: REST API
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-61325
+version: 10.20.611.0
+---
+Two new properties `operationsCreatedCount`, `operationsUpdateCount` have been added to all REST endpoints which will return detailed request counters for usage statistics.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md]:
```suggestion
Two new properties, `operationsCreatedCount` and `operationsUpdateCount`, have been added to all REST endpoints which will return detailed request counters for usage statistics. 
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@martin-c8y Please approve

**Review by martin-c8y — APPROVED** [review]

**Pull request merged**